src/index.css

```diff
@@ -1,20 +1,215 @@
+  background-color: #f7fafc;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+header {
+  display: flex;
+  background-color: white;
+  padding: 30px;
+  padding-left: 15%;
 }
 
-header{
+.Bookstore-CMS {
+  width: 260px;
+  height: 37px;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 28px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.BOOKS {
+  width: 59px;
+  height: 16px;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 13px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  padding-top: 15px;
+  padding-left: 10px;
+}
+
+.CATEGORIES {
+  width: 105px;
+  height: 16px;
+  opacity: 0.5;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 13px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  padding-top: 15px;
+  padding-left: 10px;
+}
+
+.Oval {
+  width: 45px;
+  height: 45px;
+  border: solid 1px #e8e8e8;
+  border-radius: 50%;
+  margin-left:50%;
   text-align: center;
-  color: green;
-  font-size: 40px;
+}
+
+.fa-user {
+  font-size: 30px;
+  padding-top: 5px;
+}
+
+.views {
+  font-size: 30px;
+  padding-left: 15% ;
+  font-family: monospace;
+  color: #56a0da;
+  padding-top: 20px;
+}
+
+.filters{
+    margin-left: 20px;
+    width: 285px;
+    height: 45px;
+    border-radius: 4px;
+    border: solid 1px #e8e8e8;
+    background-color: #ffffff;
+    font-size: 18px;
+}
+
+.book {
+    margin-left: 15%;
+    margin-top: 20px;
+    background-color: white;
+    width:70%;
+    padding: 20px;
+    display: flex;
+    border-radius: 10px;
+}
+.bookCat {
+  width: 49px;
+  height: 18px;
+  opacity: 0.5;
+  font-family: Montserrat;
+  font-size: 14px;
   font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.bookName {
+  width: 400px;
+  height: 29px;
+  font-family: RobotoSlab;
+  font-size: 22px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+.bookAuthor {
+  width: 107px;
+  height: 19px;
+  font-family: RobotoSlab;
+  font-size: 14px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.removeBook {
+  margin-left: 40%;
+  background-color: #0290ff;
+  width: 200px;
+  border-radius: 20px;
+}
```

Comment:
We can remove the default button styling with `border: none;`

- Current
   ![image](https://user-images.githubusercontent.com/36057474/75662882-2b411700-5c70-11ea-8c63-89dd4c35a8ae.png)

- Suggested
   ![image](https://user-images.githubusercontent.com/36057474/75662837-12d0fc80-5c70-11ea-9860-f751e2a240da.png)